cmf-cli/resources/template_feed/init/Libs/LBOs/generateLBOs.ps1

```diff
@@ -1,23 +1,22 @@
-$invocation = (Get-Variable MyInvocation).Value
-$pwd = Split-Path $invocation.MyCommand.Path
-$lbosPath = "$PSScriptRoot\..\LBOs"
-
-# Create a new session remotely to wait for process
-$sess = New-PSSession -ComputerName "<%= $CLI_PARAM_vmHostname %>"
-Enter-PSSession -Session $sess
-Write-Host "Deal with host config file"
-
-Invoke-Command -Session $sess -Scriptblock {$protectUnprotectConfigFilePath = '<%= $CLI_PARAM_InstallationPath %>\ProtectUnprotectConfigFile'}
-Invoke-Command -Session $sess -Scriptblock {$lboGeneratorPath = '<%= $CLI_PARAM_InstallationPath %>\LBOGenerator'}
-Invoke-Command -Session $sess -Scriptblock {$pathBiz = '<%= $CLI_PARAM_InstallationPath %>\BusinessTier'}
-Invoke-Command -Session $sess -Scriptblock {$arguments = "/Mode:2 /InstallPath:""$pathBiz"""}
-Invoke-Command -Session $sess -Scriptblock {Set-Location -Path $protectUnprotectConfigFilePath}
-Invoke-Command -Session $sess -Scriptblock {Start-Process Cmf.Tools.ProtectUnprotectConfigFile.exe $arguments -Wait}
-Invoke-Command -Session $sess -Scriptblock {Set-Location -Path $lboGeneratorPath}
-Invoke-Command -Session $sess -ScriptBlock {.\LBOUpdater.ps1}
-
-Remove-PSSession -Session $sess
-Exit-PSSession
+Clear-Host
+
+# Set Variables to later copy LBO's
+$businessPath = "$PSScriptRoot\..\..\LocalEnvironment\BusinessTier"
+$masterDataPath = "$PSScriptRoot\..\..\LocalEnvironment\MasterDataLoader"
+$lbosPath = "$PSScriptRoot"
+
+
+
+# Generate LBOs
+Write-Host "Generate .Net LBOs"
+$installation = "$PSScriptRoot\..\..\LocalEnvironment"
```

Comment:
We're not maintaining this script, just fixing it so that it's not invalid when working with container deployments. The enhancement will be included in the development plugin, I don't seem much point in enhancing it here...